### ISSUE TITLE
No Connection

### ISSUE THREAD

**Issue by bannedl-media** [YoYoGames/GameMaker-Bugs#2489]
### Description

No matter what I try to do, Gamemaker refuses to connect to the servers and I'm logged out and can't sign back in, locking me out of my projects.

### Expected Change

Please let me connect again

### Steps To Reproduce

1. fwfwf

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2023.11.0 (Monthly)

### Which operating system(s) are you seeing the problem on?

Windows 10

### Are you running GameMaker from inside your Steam library?

Yes

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by stuckie** [comment]
It looks like GameMaker is being blocked on your system, which isn’t something we can fix. I will close this issue now, but please see [our guide](https://help.yoyogames.com/hc/en-us/articles/360022953052) on what permissions and antivirus settings are required for GameMaker to work properly.

We are also going to need the ui.log to see what error you are getting on startup.
Please file a new ticket with this information if the above does not help.
